server/config.ts

```diff
@@ -98,6 +98,9 @@ export default {
   analytics: {
     googleAnalyticsId: get('GOOGLE_ANALYTICS_ID', ''),
   },
+  featureFlags: {
+    showAnalytics: get('SHOW_ANALYTICS', '') === 'true',
```

Comment:
I like that this is in it's own "namespace" as we may add more feature flags.

Would be odd to prepend these (future) environment variables with something to group them together? e.g. `FF_SHOW_ANALYTICS`? Or maybe `FEATURE_SHOW_ANALYTICS`?